fiftyone/server/query.py

```diff
@@ -356,11 +356,19 @@ def saved_view(
     ) -> t.Optional[SavedView]:
         ds = fo.load_dataset(dataset_name)
         if ds.has_views & ds.has_view(view_name):
-            for view in ds._doc.saved_views:
-                if view.name == view_name:
+            for the_view_name in ds.list_views:
+                if the_view_name == view_name:
                     return view
         return
 
+    @gql.field
+    def saved_views(self, dataset_name: str) -> t.List[SavedView]:
```

Comment:
Is this necessary? The list of saved views is already added to the Dataset graphql object, so you should be able to just directly read it from the query result already

Reply:
I separated it so I can create a refetchable fragments and refresh the views separate from reloading the entire dataset.